Tests/DataPipeline/APITest.swift

```diff
@@ -110,6 +91,7 @@ class DataPipelineAPITest: UnitTest {
         try skipRunningTest()
 
         _ = SDKConfigBuilder(cdpApiKey: .random)
+            .region(.US)
```

Comment:
I think it would be better to use `Region.US` instead as `.US` doesn't require import while `Region.US` will ask for import if not already there 🤔

Reply:
> instead as .US doesn't require import while Region.US will ask for import

are you sure about this? i would have thought both are the same thing? and would require import either way? 🤔

Reply:
Discussed on slack, it does require import for `Region.US`

Reply:
I fixed this same bug a long time ago where Region was not being exposed in the Track module. 

If I remember correctly, this is the behavior that I experienced in APITest class when I fixed that bug. 

```swift
public class APITest {
  // Before I added the `Region` typelias...
  _ = SDKConfigBuilder(cdpApiKey: .random)
    .region(.US) // This would compile just fine. 
  let _: Region = .US // But this would not compile. 

  // After I added the `Region` typealias, the let statement would compile 🤷‍♀️
}
```

So, Rehan is on the right track here. But I do think the only way we can verify if `Region` is truly accessible in the public API is to try and directly reference it [like this APITest is doing](https://github.com/customerio/customerio-ios/blob/33861dbf6d2b465567472ba38e8b78aff377baca/Tests/Common/APITest.swift#L34-L36), not use the Builder function parameter.